pages/price-feeds/use-real-time-data/stacks.mdx

```diff
@@ -0,0 +1,172 @@
+
+```bash
+{
+  price-identifier: 0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43,
+  price: 10603557773590,
+  conf: u3776653890,
+  ema-price: 10602069900000,
+  ema-conf: u4062895700,
+  expo: -8,
+  publish-time: u1750425711,
+  prev-publish-time: u1750425710
+}
+```
+
+With the price feed data, we can adjust the price based on the `expo` property. Price feeds represent numbers in a fixed-point format. So in the above returned price feed data, the price of `10603557773590` and given `expo` of `-8` should be formatted as `106035`. The same exponent is used for both the price and confidence interval.
+
+We can then determine the USD amount of sBTC the user owns and decide if it is enough to mint a `benjamin-nft` for $100 worth of sBTC. Benjamin is in reference to Benjamin Franklin being the face of a one hundred dollar bill, get it?
+
+```clarity
+;; --snip --
+      ;; Price feeds represent numbers in a fixed-point format. The expo property tells us
+      ;; at what certain position is the decimal point implicity fixed.
+      (price-denomination (pow 10 (* (get expo price-data) -1)))
+      ;; We'll adjust the price to its normal decimal representation.
+      (adjusted-price (to-uint (/ (get price price-data) price-denomination)))
+      ;; Get the user's current sBTC balance.
+      (user-sbtc-balance (unwrap!
+        (contract-call? 'SM3VDXK3WZZSA84XXFKAFAF15NNZX32CTSG82JFQ4.sbtc-token
+          get-balance-available tx-sender
+        )
+        ERR_READING_SBTC_BALANCE
+      ))
+    )
+    ;; Determine if the user has at least $100 worth of sBTC to join the Benjamin Club.
+    (if (> (/ (* user-sbtc-balance adjusted-price) (to-uint price-denomination))
+        COST-OF-BENJAMIN-NFT
+      )
+      (let ((hundred-dollars-in-sbtc (/ (* COST-OF-BENJAMIN-NFT (to-uint price-denomination)) adjusted-price)))
+        (try! (contract-call? 'SM3VDXK3WZZSA84XXFKAFAF15NNZX32CTSG82JFQ4.sbtc-token
+          transfer hundred-dollars-in-sbtc tx-sender (as-contract tx-sender)
+          none
+        ))
+        (contract-call? .nft-contract mint tx-sender)
+      )
+      ERR_NOT_ENOUGH_SBTC
+    )
+  )
+)
+```
+
+<Callout type="warning" emoji="⚠️">
+The `verify-and-update-price-feeds` of the `.pyth-oracle-v3` contract applies a fee of 1 uSTX, or 1 micro-stx, which is 0.000001 STX.
+</Callout>
+
+Alternatively, developers can just invoke the `read-price-feed` public function of the `pyth-oracle-v3.clar` contract. This will simply return the price feed from the last updated feed. But it's always encouraged to determine if a staleness check is viable for your application.
+
+## Write Front-End Code
+
+In your front-end application code, you can install and use the methods brought by Pyth Network's Javascript SDK to fetch the latest price update, known as a VAA (Verified Action Approvals) message.
+
+```javascript copy
+import { PriceServiceConnection } from "@pythnetwork/price-service-client"
```

Comment:
Please replace this with the [hermes-client](https://github.com/pyth-network/pyth-crosschain/tree/main/apps/hermes/client/js)